### PR TITLE
Enable running within a child process.

### PR DESCRIPTION
**Pull request by aichholzer**
Useful for using `boxen` in child processes.

While implementing the float option on the [CLI](https://github.com/aichholzer/boxen-cli) version, I noticed the tests failing since the child process does not have any knowledge about the parent's `stdout.columns`, hence this. [CLI test case](https://github.com/aichholzer/boxen-cli/blob/master/test.js)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ module.exports = (text, opts) => {
 
 	const contentWidth = widestLine(text) + padding.left + padding.right;
 	const paddingLeft = PAD.repeat(padding.left);
-	const columns = process.stdout.columns;
+	const columns = process.env.STDOUT_COLUMNS || process.stdout.columns;
```
Comment (sindresorhus) [inline index.js]: I think we instead should just default to `80` columns when it's not available.

Comment (aichholzer) [inline index.js]: I thought about that too, but then the case of running within a child process came along, where you might still have a window wider than 80, in which case neither `left` nor `center` would work. In any case, I am merely helping out here, so whatever you consider to be the right approach is fine.

Let me know and I will update the code.

Comment (sindresorhus) [inline index.js]: Seems like `process.env.COLUMNS` is the convention for this, so use that and add `80` as last fallback:

```js
const columns =  process.stdout.columns || process.env.COLUMNS || 80;
```

**Comment by sindresorhus** [comment]
Btw, you should never do PRs from master. Then you'll end up with the commit mess like you have here ;)

File: index.js (hunk 1 above)
Comment (aichholzer) [inline index.js]: Sounds like a plan. I did not a `convention` for this particular case even existed.

**Comment by aichholzer** [comment]
Yep, indeed. I usually don't PR from master... I guess I was not fully focusing.

**Pull request merged**